testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js

```diff
@@ -1710,6 +1710,30 @@ QUnit.module('aria accessibility', moduleConfig, () => {
         assert.deepEqual(list._getAriaTarget(), dropDownList._getAriaTarget());
         assert.strictEqual($input.attr('role'), 'combobox', 'input.role');
     });
+
+    QUnit.test('list aria-label should be set correctly if items changes in runtime', function(assert) {
```

Comment:
it tests not only a runtime change
```suggestion
    QUnit.test('list aria-label should be set correctly', function(assert) {
```

Reply:
Thanks for your attention! Was fixed in [d55def719df90757723ca998b13c81a9f9a52e0b].